qtazu/widgets/login.py

```diff
@@ -1,11 +1,73 @@
 import os
 import logging
+import re
 
 import gazu
 from Qt import QtWidgets, QtGui, QtCore
 
 log = logging.getLogger(__name__)
 
+PLACEHOLDER = os.path.join(
```

Comment:
Is this better refactored as `KITSU_LOGO` as opposed to `PLACEHOLDER`? Don't really care, but placeholder does seem a bit vague.